docs/sources/datasources/tempo.md

```diff
@@ -31,6 +31,7 @@ This is a configuration for the [trace to logs feature]({{< relref "../explore/t
 
 - **Data source -** Target data source.
 - **Tags -** The tags that will be used in the Loki query. Default is `'cluster', 'hostname', 'namespace', 'pod'`.
+- **Map tag names -** Enables mapping Tempo tag names to Loki label names. For example, map `service.name` to `service`.
```

Comment:
```suggestion
- **Map tag names -** Maps the Tempo tag names to Loki label names. For example, map `service.name` to `service`.
```

Reply:
Given that 'Map tag names' is a toggle, would 'Enables ...' be the correct wording? This option doesn't actually do the mapping, it just turns it on/off.

Reply:
How about: "When enabled, map the Tempo tag names to Loki label names. For example, map `service.name` to `service`."

Reply:
I would mention that this is based on user configuration. To me, it reads a bit like tags are automatically mapped. For example, it could be: "When enabled, allows configuring how Tempo tag names are mapped to Loki label names. For example, map service.name to service."